slices/apparmor.yaml

```diff
@@ -0,0 +1,199 @@
+      /etc/apparmor.d/mmdebstrap:
+      /etc/apparmor.d/msedge:
+      /etc/apparmor.d/nautilus:
+      /etc/apparmor.d/notepadqq:
+      /etc/apparmor.d/nvidia_modprobe:
+      /etc/apparmor.d/obsidian:
+      /etc/apparmor.d/opam:
+      /etc/apparmor.d/opera:
+      /etc/apparmor.d/pageedit:
+      /etc/apparmor.d/plasmashell:
+      /etc/apparmor.d/podman:
+      /etc/apparmor.d/polypane:
+      /etc/apparmor.d/privacybrowser:
+      /etc/apparmor.d/qcam:
+      /etc/apparmor.d/qmapshack:
+      /etc/apparmor.d/qutebrowser:
+      /etc/apparmor.d/rootlesskit:
+      /etc/apparmor.d/rpm:
+      /etc/apparmor.d/rssguard:
+      /etc/apparmor.d/runc:
+      /etc/apparmor.d/sbuild:
+      /etc/apparmor.d/sbuild-abort:
+      /etc/apparmor.d/sbuild-adduser:
+      /etc/apparmor.d/sbuild-apt:
+      /etc/apparmor.d/sbuild-checkpackages:
+      /etc/apparmor.d/sbuild-clean:
+      /etc/apparmor.d/sbuild-createchroot:
+      /etc/apparmor.d/sbuild-destroychroot:
+      /etc/apparmor.d/sbuild-distupgrade:
+      /etc/apparmor.d/sbuild-hold:
+      /etc/apparmor.d/sbuild-shell:
+      /etc/apparmor.d/sbuild-unhold:
+      /etc/apparmor.d/sbuild-update:
+      /etc/apparmor.d/sbuild-upgrade:
+      /etc/apparmor.d/scide:
+      /etc/apparmor.d/signal-desktop:
+      /etc/apparmor.d/slack:
+      /etc/apparmor.d/slirp4netns:
+      /etc/apparmor.d/steam:
+      /etc/apparmor.d/stress-ng:
+      /etc/apparmor.d/surfshark:
+      /etc/apparmor.d/systemd-coredump:
+      /etc/apparmor.d/thunderbird:
+      /etc/apparmor.d/toybox:
+      /etc/apparmor.d/trinity:
+      /etc/apparmor.d/tunables/alias:
+      /etc/apparmor.d/tunables/apparmorfs:
+      /etc/apparmor.d/tunables/dovecot:
+      /etc/apparmor.d/tunables/etc:
+      /etc/apparmor.d/tunables/global:
+      /etc/apparmor.d/tunables/home:
+      /etc/apparmor.d/tunables/home.d/site.local:
+      /etc/apparmor.d/tunables/home.d/ubuntu: {text: FIXME, mutable: true}
+      /etc/apparmor.d/tunables/home.d/ubuntu.tmp:
+        text: |
+          # This file is auto-generated. It is recommended you update it using:
+          # $ sudo dpkg-reconfigure apparmor
+          #
+          # The following is a space-separated list of where additional user home
+          # directories are stored, each must have a trailing '/'. Directories added
+          # here are appended to @{HOMEDIRS}.  See tunables/home for details.
+          #@{HOMEDIRS}+=
+        until: mutate
+      /etc/apparmor.d/tunables/kernelvars:
+      /etc/apparmor.d/tunables/multiarch:
+      /etc/apparmor.d/tunables/multiarch.d/site.local:
+      /etc/apparmor.d/tunables/proc:
+      /etc/apparmor.d/tunables/run:
+      /etc/apparmor.d/tunables/securityfs:
+      /etc/apparmor.d/tunables/share:
+      /etc/apparmor.d/tunables/sys:
+      /etc/apparmor.d/tunables/xdg-user-dirs:
+      /etc/apparmor.d/tunables/xdg-user-dirs.d/site.local: {text: FIXME, mutable: true}
+      /etc/apparmor.d/tunables/xdg-user-dirs.d/site.local.tmp:
```

Comment:
hum do you really need a .tmp? I know that's what the maintainer script does, but I don't even understand why...? The script creates the tmpfile, write to it, and immediately moves it to `site.local`...so why not write in there directly and avoid the mutation script?

Reply:
Good point!